generators/Readme.md

```diff
@@ -64,23 +64,22 @@ A generator gets passed in a JSON blob including the following information:
 1. Everything which is declared in the `schema.prisma` file:
 
    - Models
-
    - Enums
    - Datasources
    - Generators, including the configuration this generator points to
 
 2. The available Query Schema of the Query Engine
 
-The following TypeScript type definition `GeneratorOptions` describes the content of the JSON blob:
+The following TypeScript type definition `GeneratorOptions` describes the content of that JSON blob:
 
 ```ts
 type GeneratorConfig = {
   output: string | null
   name: string
   provider: string
   config: Dictionary<string>
-  platforms: string[]
-  pinnedPlatform?: string | null
+  binaryTargets: string[]
+  pinnedBinaryTarget?: string | null
```

Comment:
This relates to some other spec (that was recently changed)? If so, linking there would be smart here.